sunpy/map/maputils.py

```diff
@@ -404,14 +404,13 @@ def contains_coordinate(smap, coordinates):
         returns a boolean arrary.
     """
     # Dimensions of smap
-    xs, ys = smap.dimensions
+    ys, xs = smap.wcs.array_shape
     # Converting coordinates to pixels
-    xc, yc = smap.world_to_pixel(coordinates)
-    point5pix = 0.5 * u.pix
-    return ((xc >= -point5pix) &
-            (xc <= xs - point5pix) &
-            (yc >= -point5pix) &
-            (yc <= ys - point5pix))
+    xc, yc = smap.wcs.world_to_pixel(coordinates)
+    return ((xc >= -0.5) &
```

Comment:
Hasn't this changed the return from being Quantity with units of pix to floats?

Reply:
The return is `bool` no? (adding this to my log of "reasons to add typing" 😉 )

Reply:
Yes it returns a boolean array

Reply:
I mean I could have read the docstring as well :rofl: